js/index.js

```diff
@@ -1 +1,120 @@
 // Your code goes here
+// prevent default
+
+const atags = document.getElementsByClassName('nav');
+console.log(atags)
+for (var i = 0 ; i < atags.length; i++) {
+    atags[i].addEventListener('click' , function(event){
```

Comment:
For line 4, you might have meant `"nav-link"` for your class name - `"nav"` actually points to your `<nav>` element (rather than your `<a>` elements), and this causes the `click` event to fire even when clicking on the space between two links.